src/frontend/screens/Settings/components/ExperimentalFeatures.tsx

```diff
@@ -4,13 +4,13 @@ import useSetting from 'frontend/hooks/useSetting'
 import { ToggleSwitch } from 'frontend/components/UI'
 import ContextProvider from 'frontend/state/ContextProvider'
 
-const FEATURES = ['enableNewDesign']
+const FEATURES = ['enableNewDesign', 'automaticWinetricksFixes']
 
 const ExperimentalFeatures = () => {
   const { t } = useTranslation()
   const [experimentalFeatures, setExperimentalFeatures] = useSetting(
     'experimentalFeatures',
-    { enableNewDesign: false }
+    { enableNewDesign: false, automaticWinetricksFixes: false }
```

Comment:
Will this be shown on Windows?

Reply:
yeah, this is displayed on windows

it doesn't really do anything on windows, but I'll update it to not show the option to avoid confusing the user